homeassistant/components/sutro/en.json

```diff
@@ -0,0 +1,25 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Sutro",
+        "description": "If you need help with the configuration have a look here: https://github.com/ydogandjiev/hass-sutro",
+        "data": {
+          "username": "Token"
+        }
+      }
+    },
+    "error": {
+      "auth": "Token is invalid or expired."
+    }
+  },
+  "options": {
+    "step": {
+      "user": {
+        "data": {
+          "sensor": "Sensor enabled"
+        }
+      }
+    }
+  }
```

Comment:
This should not be needed, as Home Assistant (out of the box) already provides tools for users to enable/disable sensors.